generator.yaml

```diff
@@ -7,11 +7,28 @@ ignore:
     # FunctionUrlConfig
     # LayerVersion
 operations:
-  PublishLayerVersion:
+  PublishLayerVersion: #opID
```

Comment:
Is this comment needed or relevant? (can we remove it?)

Reply:
My mistake, have removed it now. Thanks!